docs/bgp-policy.md

```diff
@@ -81,8 +81,12 @@ as the effective BGPPolicy.
 
 ### LocalASN
 
-The `localASN` field defines the Autonomous System Number (ASN) that the local BGP process uses. The available private
-ASN range is `64512-65535`. The field is mandatory.
+The `localASN` field defines the Autonomous System Number (ASN) that the local BGP process uses. This field is mandatory
+and accepts values in the range of `1-4294967295`.
+
+For private ASNs, which are within the ranges 64512-65534 (16-bit) or 4200000000-4294967294 (32-bit), ensure they are
+used only within private networks or scenarios where they will not conflict with public ASNs. If public network
```

Comment:
I don't quite understand how this happens: "private ASNs within the ranges 64512-65534 (16-bit) or 4200000000-4294967294 (32-bit)" conflict with public ASNs, aren't they in different ranges?

Reply:
+1, after we extend the bits to 32, I suppose we only support 4200000000-4294967294 (32-bit) as the private ASN?  or it's user's decision to define which are the private ASN?

Reply:
Will rephrase this sentence.

Reply:
Done